src/components/forms/FamilyForm.tsx

```diff
@@ -205,7 +223,7 @@ export const FamilyForm = ({ family: loadedFamily }: TProps) => {
       // We still expect this value in the backend
       geography: formData.geographies?.[0].value || '',
       geographies: formData.geographies?.map((geo) => geo.value),
-      category: isMCFCorpus ? 'MCF' : formData.category,
+      category: formData.category,
```

Comment:
Question: just for visibility - how or why are we able to remove this default now?